src/Grammar.cpp

```diff
@@ -0,0 +1,36 @@
+#include "Grammar.h"
+#include <fstream>
+
+Grammar::Grammar() {
+    type = NONE;
+    content = "";
+}
+
+Grammar::Grammar(const GrammarType type, const std::string content) {
+    this->type = type;
+    this->content = content;
+
```

Comment:
We don't, this was leftover from an earlier version where it was loaded from file at a different point, to avoid doing it multiple times. Thanks, changed!

Reply:
correct